patient-case-summary/app.py

```
@@ -0,0 +1,346 @@
+                    elif ">> Guideline recommendation:" in event.msg:
+                        # Extract JSON part
+                        json_match = re.search(r'>> Guideline recommendation: ({.*})', event.msg)
+                        if json_match:
+                            try:
+                                json_data = json.loads(json_match.group(1).replace("'", '"'))
+                                log_entry = {
+                                    "type": "json",
+                                    "title": ">> Guideline recommendation:",
+                                    "data": json_data
+                                }
+                            except:
```

Ruff (0.8.2)

Do not use bare `except`

(E722)